daphne_worker/Cargo.toml

```diff
@@ -40,7 +40,7 @@ tracing-subscriber = {version = "0.3.17", features = ["env-filter", "json"]}
 url = { version = "2.3.1", features = ["serde"] }
 serde_json = "1.0.96"
 serde-wasm-bindgen = "0.5.0"
-worker = "0.0.16"
+worker = "0.0.17"
```

Comment:
Is bumping the `worker` crate necessary in order to use new features? If not, then I'd prefer to bump the version in a separate PR. The reason is that in the past we've seen bugs in `worker` releases that only manifested in the real Workers runtime (not when run locally via `wrangler dev`). Ideally we would test updates to `worker` in a live deployment before merging.

Reply:
we can do it in a separate PR :+1: